tests/integration/ceph_csi_test.go

```diff
@@ -228,10 +278,9 @@ spec:
 	if !isPodRunning {
 		k8sh.PrintPodDescribe(namespace, csiTestCephFSPodName)
 		k8sh.PrintPodStatus(namespace)
-	} else {
-		// cleanup the pod and pv
-		err = k8sh.ResourceOperation("delete", pod)
 	}
+	// cleanup the pod and pvc
+	err = k8sh.ResourceOperation("delete", pod)
```

Comment:
You could add an `assert.NoError(s.T(), err)`

Reply:
done